seqr/views/apis/staff_api_tests.py

```diff
@@ -195,3 +202,75 @@ def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
                                                       h="index,docs.count,store.size,creation.date.string")
         mock_es_client.cat.aliases.assert_called_with(format="json", h="alias,index")
         mock_get_mapping.assert_called_with(doc_type='variant,structural_variant')
+
+    def test_mme_details(self):
+        url = reverse(mme_details)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['metrics', 'genesById', 'submissions'])
+
+    def test_seqr_stats(self):
+        url = reverse(seqr_stats)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualCount', 'familyCount', 'sampleCountByType'])
+
+    def test_get_projects_for_category(self):
+        url = reverse(get_projects_for_category, args=[PROJECT_CATEGRORY_NAME])
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['projectGuids'])
+
+    def test_discovery_sheet(self):
+        non_project_url = reverse(discovery_sheet, args=[NON_PROJECT_GUID])
+        _check_login(self, non_project_url)
+
+        response = self.client.get(non_project_url)
+        self.assertEqual(response.status_code, 500)
```

Comment:
500s are unhandled errors and we don't generally test them. If there is a specific error handled in the code we want to test it would be a 400 error and we would also often want to rest the error message

Reply:
The current code raises an exception without handling it when it receives a bad project guid. So it causes an 500 status response.  Should the code be changed to generate a 400s error at this case? I leave it unchanged for now.

Reply:
Yes please update it to be a 400, and update the test to test the error message

Reply:
Updated